src/mpd.c

```diff
@@ -2721,6 +2721,27 @@ mpd_command_findadd(struct evbuffer *evbuf, int argc, char **argv, char **errmsg
   return 0;
 }
 
+/*
+ * Some MPD clients crash if the tag value includes the newline character.
+ * While they should normally not be included in most ID3 tags, they sometimes
+ * are, so we just change them to space. See #1613 for more details.
+ */
+
+static void
+sanitize_value(char **strval)
+{
+  char *ptr = *strval;
```

Comment:
Maybe 
```cpp
static void sanitize_value(char *ptr) {
   while (ptr = strchr(ptr, '\n')) 
      *ptr++ = ' ';
}
[...]
   sanitize_value(*strval);
```

is better because it's using standard C library and it's shorter ?